react/features/base/lib-jitsi-meet/actions.js

```diff
@@ -12,7 +12,7 @@ import {
     LIB_INIT_ERROR,
     LIB_WILL_DISPOSE,
     LIB_WILL_INIT
-} from './actionTypes';
+} from './actionTypes.ts';
```

Comment:
I thought we would not need this?

Reply:
If I remove this, the build fails.

Reply:
Then we need to see why, the extension should not be needed at all.

Reply:
Updated. I think the problem was in an another place.